src/vulkan-renderer/application.cpp

```diff
@@ -380,16 +380,16 @@ VkResult Application::init() {
 #endif
     }
 
-    spdlog::debug("Checking for '-novalidation' command line argument.");
+    spdlog::debug("Checking for '--no-validation' command line argument.");
```

Comment:
Instead of `Checking for` better log later `--no-validation with True` or `--no-validation not found using False`.